Formula/kafkacat.rb

```diff
@@ -12,12 +12,14 @@ class Kafkacat < Formula
     sha256 "e09845976996cd838656e0065309e06f65e8446e1f0eb01f471bda2da36553ed" => :sierra
   end
 
-  depends_on "librdkafka"
```

Comment:
Why remove this?

Reply:
So - since I've added libSerdes which requires it there, I though explicitly mentioning the dependency here is redundant.  @SMillerDev Is that considered a bad practice?

Reply:
@SMillerDev LMK if I should explicitly put back the dependency even though it's now included transitively.  I understand how you might make a case for that.  Resolving the thread for now.